src/demo/index.ejs

```diff
@@ -7,8 +7,16 @@
     <meta name="mobile-web-app-capable" content="yes">
     <meta name="apple-mobile-web-app-capable" content="yes">
     <meta name="format-detection" content="telephone=no">
-
-    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600" rel="stylesheet">
+    <% if (process.env.NODE_ENV !== "development"){ %>
+    <meta http-equiv="Content-Security-Policy" content="
+      default-src 'self' https://assets.onfido.com;
+      script-src 'unsafe-eval' 'self' https://www.woopra.com https://assets.onfido.com;
```

Comment:
I'd remove the unsafe-eval here based on the suggestion from csp-evaluator and use "strict-dynamic" with nonces.